fix_pthread.patch

```diff
@@ -0,0 +1,11 @@
+diff --unified --recursive --text franka_ros-release-release-melodic-franka_control-0.6.0-1/CMakeLists.txt franka_ros-release-release-melodic-franka_control-0.6.0-1.new/CMakeLists.txt
+--- franka_ros-release-release-melodic-franka_control-0.6.0-1/CMakeLists.txt	2019-07-11 09:54:40.000000000 +0200
```

Comment:
I dont think this patch works. The file name to be patched has to be the same.

Reply:
Yeah forgot to remove the patch it was not used, fixed.

Reply:
@liborw sorry for not personally telling you this but we have shifted our operations over to the [ros-noetic-arch](https://github.com/ros-noetic-arch) github org. See: https://discourse.ros.org/t/who-is-using-ros-on-archlinux/388/66?u=acxz
Feel free to open up the package requests there